Jenkinsfile

```diff
@@ -490,51 +557,12 @@ pipeline {
                                 '$BUILDARGS_QB_CHECK' +
                                 ' --build-arg QUICKBUILD_DEPS="' +
                                   env.QUICKBUILD_DEPS_EL7 + '"' +
-                                ' --build-arg REPOS="' + component_repos() + '"'
+                                ' --build-arg REPOS="' + pr_repos() + '"'
```

Comment:
Without my above suggestion about making `pr_repos()` return both the all-distro and per-distro values, this will be missing any `PR-repos-el7` values won't it?